api/server.py

```diff
@@ -141,6 +146,31 @@ def get_all_companies():
     return jsonify({'All Companies' : output})
 
 
+@app.route('/api/projects/delete', methods=['DELETE'])
+def delete_project():
+    projects = mongo.db.projects
+
+    project_name = request.json['project_name']
+    plaintext = request.json['password']
+    passwd = hashlib.md5(plaintext.encode('utf-8')).hexdigest()
+
+    if passwd == password:
```

Comment:
@kjeffc Don't worry about check authentication and authorization here.

@jsfleming when you build the sponsor and admin login systems, can you go through each of the routes and add the same auth check to make sure either admin or sponsor is logged in and it's the correct role for the route? (e.g. Deleting a project is reserved for admin role)